enable/wx/base_window.py

```diff
@@ -215,37 +215,37 @@ def __init__(self, parent, wid=-1, pos=wx.DefaultPosition,
 
         return
 
-    def _create_control(self, parent, wid, pos = wx.DefaultPosition,
-                        size = wx.DefaultSize):
-        return wx.Window(parent, wid, pos, size, style = wx.CLIP_CHILDREN |
-                           wx.WANTS_CHARS)
-
+    def _create_control(self, parent, wid, pos=wx.DefaultPosition, size=wx.DefaultSize):
+        if parent is None:
+            # pab hack in order to avoid that the wx.Window crash!
+            parent = wx.Frame(None)
+        return wx.Window(parent, wid, pos, size, style=wx.CLIP_CHILDREN | wx.WANTS_CHARS)
 
     def _on_close(self, event):
         # Might be scrollbars or other native components under
         # us that are generating this event
         if event.GetWindow() == self.control:
             self._gc = None
-            wx.EVT_ERASE_BACKGROUND(self.control, None)
-            wx.EVT_PAINT(self.control, None)
-            wx.EVT_SIZE(self.control, None)
-            wx.EVT_LEFT_DOWN(self.control, None)
-            wx.EVT_LEFT_UP(self.control, None)
-            wx.EVT_LEFT_DCLICK(self.control, None)
-            wx.EVT_MIDDLE_DOWN(self.control, None)
-            wx.EVT_MIDDLE_UP(self.control, None)
-            wx.EVT_MIDDLE_DCLICK(self.control, None)
-            wx.EVT_RIGHT_DOWN(self.control, None)
-            wx.EVT_RIGHT_UP(self.control, None)
-            wx.EVT_RIGHT_DCLICK(self.control, None)
-            wx.EVT_MOTION(self.control, None)
-            wx.EVT_ENTER_WINDOW(self.control, None)
-            wx.EVT_LEAVE_WINDOW(self.control, None)
-            wx.EVT_MOUSEWHEEL(self.control, None)
-            wx.EVT_KEY_DOWN(self.control, None)
-            wx.EVT_KEY_UP(self.control, None)
-            wx.EVT_CHAR(self.control, None)
-            wx.EVT_WINDOW_DESTROY(self.control, None)
+            self.control.Bind(wx.EVT_ERASE_BACKGROUND, None)
+            self.control.Bind(wx.EVT_PAINT, None)
+            self.control.Bind(wx.EVT_SIZE, None)
+            self.control.Bind(wx.EVT_LEFT_DOWN, None)
+            self.control.Bind(wx.EVT_LEFT_UP, None)
+            self.control.Bind(wx.EVT_LEFT_DCLICK, None)
+            self.control.Bind(wx.EVT_MIDDLE_DOWN, None)
+            self.control.Bind(wx.EVT_MIDDLE_UP, None)
+            self.control.Bind(wx.EVT_MIDDLE_DCLICK, None)
+            self.control.Bind(wx.EVT_RIGHT_DOWN, None)
+            self.control.Bind(wx.EVT_RIGHT_UP, None)
+            self.control.Bind(wx.EVT_RIGHT_DCLICK, None)
+            self.control.Bind(wx.EVT_MOTION, None)
+            self.control.Bind(wx.EVT_ENTER_WINDOW, None)
+            self.control.Bind(wx.EVT_LEAVE_WINDOW, None)
+            self.control.Bind(wx.EVT_MOUSEWHEEL, None)
+            self.control.Bind(wx.EVT_KEY_DOWN, None)
+            self.control.Bind(wx.EVT_KEY_UP, None)
+            self.control.Bind(wx.EVT_CHAR, None)
+            self.control.Bind(wx.EVT_WINDOW_DESTROY, None)
```

Comment:
Does binding to None mean the same as `Unbind`, e.g.?:
```
self.control.Unbind(wx.EVT_WINDOW_DESTROY)
```
`Bind` to `None` looks weird to me, just wanted to mention I am unsure what is right here.

Reply:
No idea. I have little to no experience writing Wx code

Reply:
wxPython confirms this, so we are good:

> handler – A callable object to be invoked when the event is delivered to self. Pass None to disconnect an event handler.

(from https://wxpython.org/Phoenix/docs/html/wx.EvtHandler.html#wx.EvtHandler.Bind)

Reply:
Thanks for investigating